packages/create-plugin/src/constants.ts

```diff
@@ -95,6 +95,7 @@ export const TEXT = {
   updateNpmDependenciesPrompt: '**Would you like to update the following dependencies in the `package.json?`**',
   updateNpmDependenciesSuccess: 'Successfully updated the NPM dependencies.',
   updateNpmDependenciesAborted: 'No NPM dependencies have been updated.',
+  updateGrafanaNpmDependenciesPrompt: '**Would you like to update the Grafana packages (@grafana/*) in the `package.json?`**',
```

Comment:
Given @jackw 's comments in the issue:
> This could introduce painful upgrade paths for devs related to type changes or api changes when they only want to keep their configs up to date

It feels like we'd want the default behaviour to be 'no'? Or at least to advise on the implications of updating these?

Reply:
I pushed an update to always skip the dependencies. I agree that's a better default behaviour.